### PR TITLE
Some UI improvements

### PR DESCRIPTION
**Pull request by barcia**
* Add border radius to Feature Talks and fix background
* Add Navbar visibility on scroll
* Add marquee
* Fix sima images aspect ratio

### CODE REVIEW

File: frontend/src/lib/Header.svelte [hunk 1]
```diff
@@ -11,16 +13,49 @@
...
+    }
+
     .navbar-center {
         width: 100%;
         justify-content: center;
     }
 </style>
 
-<nav class="navbar is-dark is-spaced" aria-label="main-navigation">
+<nav class="navbar is-dark is-spaced {navbarHiddenClass} " aria-label="main-navigation">
```
Comment (roberto-red) [inline frontend/src/lib/Header.svelte]: You may want to use [the `class` element directive here](https://svelte.dev/docs/element-directives#class-name) for clarity:
```html
<nav class="navbar is-dark is-spaced" class:is-concealed={isNavbarHidden} aria-label="main-navigation">
```

File: frontend/src/routes/+page.svelte [hunk 2]
```diff
@@ -97,11 +106,8 @@ import { onMount } from 'svelte';
 </section>
 
 <aside class="has-background-primary-light">
-    <div class="level is-mobile text--small--uppercase py-4">
-        {#each Array(16) as _}
-        <div class="level-item">Registration is open</div>
-        <div class="level-item">🔥</div>
-        {/each}
+    <div id='marquee' class="marquee level is-mobile text--small--uppercase py-4">
+        <div>Registration is open&nbsp;🔥&nbsp;</div>
```
Comment (roberto-red) [inline frontend/src/routes/+page.svelte]: This text is provided by the user on the CMS, so we can't just rely on them using non-breaking spaces. A margin-left will serve, maybe? If it's absolutely necessary, we can just replace all spaces of the string with non-breaking ones and append an extra space at the end:

```js
marqueeText.trim().replace(/\s/g, '\xa0') + '\xa0'
```

**Review by roberto-red — APPROVED** [review]
Well, this is nice!

Please, take a look at the two comments if you have the time.

File: frontend/src/lib/Header.svelte (hunk 1 above)
Comment (barcia) [reply frontend/src/lib/Header.svelte]: Done!

File: frontend/src/routes/+page.svelte (hunk 2 above)
Comment (barcia) [reply frontend/src/routes/+page.svelte]: Fixed! Now, I use a `1ch` padding right for each element. This makes one character space between each element. I think that there is not one _technical solution_ for that, is a design solution.

**Pull request merged**